packages/sparrowql-graphql/src/graph.ts

```diff
@@ -129,10 +156,15 @@ export function astToOptions(
 
 export function astToPipeline(info: GraphQLResolveInfo, options: Options) {
   const scopes: string[] = [];
-  const inflateMap: Record<string, unknown> = {};
-  const projection: Record<string, unknown> = {};
+  const inflateMap: Record<string, ProjectionType | string> = {};
+  const projection: ProjectionType = {};
+
+  const queryType = info.schema.getQueryType();
+  if (!queryType) {
+    throw new Error('Missing Query object in schema.');
+  }
 
-  const rootType = extractType(info.schema.getQueryType()!);
+  const rootType = extractType(queryType);
   const extractPathType = (path: string[]) =>
     (path.reduce(
```

Comment:
One of the last two problematic which I can't seem to figure out. Would you mind explaining a little bit what is happening here? Are we trying to find the nested node in the schema? @radekmie

Reply:
Exactly, it's basically `_.get(schema, name): GraphQLType`.